### PR TITLE
Prae acid spray buff

### PR DESCRIPTION
**Pull request by goose4429**
## About The Pull Request

Decreases slow after use.

## Why It's Good For The Game

Noob trap. No reason for it to be in the game really, but MJP doesn't want it gone.

## Changelog
:cl:
balance: Prae Spray slow 1 -> 0.5
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/praetorian/abilities_praetorian.dm [hunk 1]
```diff
@@ -103,7 +102,7 @@ GLOBAL_LIST_INIT(acid_spray_hit, typecacheof(list(/obj/structure/barricade, /obj
 			is_blocked = TRUE
 	if(!is_blocked)
 		if(!skip_timer)
-			addtimer(CALLBACK(src, .proc/continue_acid_cone_spray, T, next_normal_turf, distance_left, facing, direction_flag, spray), 3)
+			addtimer(CALLBACK(src, .proc/continue_acid_cone_spray, T, next_normal_turf, distance_left, facing, direction_flag, spray), 2)
```
Comment (Vallat) [inline code/modules/mob/living/carbon/xenomorph/castes/praetorian/abilities_praetorian.dm:105]:
```suggestion
			addtimer(CALLBACK(src, .proc/continue_acid_cone_spray, T, next_normal_turf, distance_left, facing, direction_flag, spray), 0.2 SECONDS)
```

**Comment by usnpeepoo** [comment]
prae is already powerful enough, removing the slow is the only thing that should be acceptable

**Comment by Naaanii** [comment]
Double buff isn't good for an already decent caste. Agree with the above comment, removing slowdown would be nice.

A big aoe knockdown doesn't really need to be faster as the slight warning sign and windup allows people to try and dodge it but a Prae in range can also practically make it undodgeable so a faster spray is really not needed.

**Comment by goose4429** [comment]
> Double buff isn't good for an already decent caste. Agree with the above comment, removing slowdown would be nice.
> 
> A big aoe knockdown doesn't really need to be faster as the slight warning sign and windup allows people to try and dodge it but a Prae in range can also practically make it undodgeable so a faster spray is really not needed.

Making it less dodgeable is the purpose of PR.
Dodging shotgun PB or a rocket is pretty hard, you know? But for spit, you just walk away, and it literally allows you to outrun it.

**Comment by Naaanii** [comment]
> > Double buff isn't good for an already decent caste. Agree with the above comment, removing slowdown would be nice.
> > A big aoe knockdown doesn't really need to be faster as the slight warning sign and windup allows people to try and dodge it but a Prae in range can also practically make it undodgeable so a faster spray is really not needed.
> 
> Making it less dodgeable is the purpose of PR. Dodging shotgun PB or a rocket is pretty hard, you know? But for spit, you just walk away, and it literally allows you to outrun it.

Shotgun PB isn't really area denial CC so I don't understand the comparison, rockets are a one shot burst damage sort of deal. Only light armour can actually outrun a spray effectively, and if hit by the spray, they're most likely going to die for wearing light armour, and like I said, a well placed and timed spray is undodgeable especially in cqc which Xenos excel at. Making it faster than all other sprays just makes it overtuned and without the slowdown, the double buff just makes Prae way too strong at range and CQC (which it already is). Added speed also kind of removes the point of the warning icon and leaves little counter play to the ability other than just predicting or using future vision.

**Comment by Losenis** [comment]
>Imagine standing still, open to stagger, for almost a second for spew to actually start moving towards marines, only to see them casually walking away faster than spit covers the ground

Either you:
Sprayed at max range
Sprayed alone at marines expecting it
Sprayed out in the open without obstructions for marines
Combination of the above

Spray can absolutely massacre marines already in the middle of a fight given it's not only a stun but also a constant source of burn damage that cover several tiles, and in that specific case it being burn damage is actually worse for marines given it causes more pain and will weaken/crit the marines far faster than the brute damage the other xenos would be inflicting.

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Double buff definitely not gonna be accepted

**Comment by goose4429** [comment]
Apparently double buffs are not accepted. Reverted speed buff.

Funny:
Quite literally searched "buff" and half of actual buffs are double buffs.
https://github.com/tgstation/TerraGov-Marine-Corps/pull/10499
https://github.com/tgstation/TerraGov-Marine-Corps/pull/9695
https://github.com/tgstation/TerraGov-Marine-Corps/pull/9578

**Comment by Naaanii** [comment]
> Apparently double buffs are not accepted. Reverted speed buff.
> 
> Funny: Quite literally searched "buff" and half of (relevant) merged buffs are double buffs. #10499 #9695 #9578

2 were features in a not so great spot or severely unused and the AMR got completely reverted in the end.

Prae is already in a good spot and doesn't even require buffs but slowdown removal sounds like a good idea with stuns being tweaked.

**Comment by goose4429** [comment]
Funny 20 days old one-line PR

**Pull request closed without merging**